packages/lib-react-components/src/MetaToolsButton/MetaToolsButton.js

```diff
@@ -6,8 +6,13 @@ import PlainButton from '../PlainButton'
 export const StyledPlainButton = styled(PlainButton)`
   > div {
     justify-content: flex-start;
+    ${props => css`padding: ${props.padding};`}
```

Comment:
Awesome! This was indeed missing, so the changes make more sense! Non-blocking suggestion, but the `MetaToolsButton` is missing a storybook story, readme, etc. Perhaps in a follow up PR, a story and a readme documenting its function and the props it accepts would be helpful. Thanks!

Reply:
If it's missing a story and a bit more, I'll save this for a separate PR.